app/models/BasicPokemonDetail.js

```diff
@@ -9,6 +9,182 @@ let BasicPokemonDetailSchema = new mongoose.Schema({
     },
     name: {
         type: String
+    },
+    gender: {
+        /*male: {
```

Comment:
I also thought about that but boolean data are faster and easier to read. I mean the data are only relevant for the machine learning algorithms and instead of searching for strings which have to be brought in the correct form and where errors are more likely to appear when something is not correctly written the guys just have to execute some if statements. So I would make booleans everywhere it is possible.